test/components/interpolatedColorLegendTests.ts

```diff
@@ -270,219 +243,198 @@ describe("InterpolatedColorLegend", () => {
   });
 
   it("has more swatches than default when expand is true", () => {
```

Comment:
Probably this test can be replaced now that we use a gradient, but replacement probably out of scope for this PR.